core/storage/user/gae_models.py

```diff
@@ -32,6 +32,9 @@ class UserSettingsModel(base_models.BaseModel):
     """
     # Email address of the user.
     email = ndb.StringProperty(required=True, indexed=True)
+    # User role. Required for authorization. EXPLORATION_EDITOR is default
```

Comment:
Is it necessary to mention the default here? The defaulting isn't done in the storage layer.